Add CHANGELOG.md for TouchFree and Web Tooling

Craig-J opened the pull request:
## Summary

_Summary of the purpose of this pull request._

Add a changelog to track changes for both:

- TouchFree - documents changes to all components released together in the main bundle with the touchfree installer
- TouchFree Web Tooling - documents changes to the web tooling which is released separately to the above

All historical releases added to the changelogs already.

### Motivation

Intent is to enhance our workflow when dealing with the internal release notes page on confluence.
Developers will manage software changes and document them in the changelog, which in turn can be refined into product release notes on the release info pages.
Changelogs are intended to replace the release notes field on Jira tickets which have not been getting used consistently.

- With a changelog in the repo we can review it closely with contributions to the code itself during PRs.
- Changes for a release are built up naturally as we merge rather than requiring collation from release notes when performing the release process.
- CHANGELOG.md is a common practice by adopted by many projects. This makes it naturally discoverable.
- CHANGELOG.md can be distributed directly with the release, allowing changes to be viewed offline.

Craig-J committed:
Add CHANGELOG.md to web tooling

Craig-J committed:
WIP

Craig-J committed:
Merge remote-tracking branch 'origin/develop' into feature/web-tooling-changelog

Craig-J committed:
Update changelogs

Craig-J committed:
Merge remote-tracking branch 'origin/develop' into feature/web-tooling-changelog

JordanBerni-UL commented on TF_Tooling_Web/CHANGELOG.md:
```diff
@@ -0,0 +1,63 @@
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+[docs-website]: https://docs.ultraleap.com/touchfree-user-manual/ "Ultraleap Docs"
```
👀 This line doesn't appear when viewing the page

JordanBerni-UL replied:
![image](https://user-images.githubusercontent.com/107030829/212857312-1f243184-0b26-4642-8413-33ca452f9eb3.png)

Craig-J replied:
Replaced with some badges at the top (with discord and support email too)

JordanBerni-UL commented on CHANGELOG.md:
```diff
@@ -0,0 +1,141 @@
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+[docs-website]: https://docs.ultraleap.com/touchfree-user-manual/ "Ultraleap Docs"
```
👀 Line missing (maybe because the space in the URL?)

Craig-J replied:
Replaced with some badges at the top (with discord and support email too)

Craig-J committed:
Add discord, docs and support badges

Craig-J committed:
Add brightsign changelog, update windows one

JordanBerni-UL reviewed (changes requested):
Removing approval as want to add changelog for brightsign

Craig-J committed:
Simplify tracking service notes in changelogs

Craig-J committed:
Merge remote-tracking branch 'origin/develop' into feature/web-tooling-changelog

JordanBerni-UL reviewed (commented):
Windows changelog is missing https://github.com/ultraleap/TouchFree/pull/410

BRS changelog is missing https://github.com/ultraleap/TouchFree/pull/412

JordanBerni-UL commented on .github/PULL_REQUEST_TEMPLATE/lightweight.md:
```diff
@@ -19,6 +19,7 @@ _Summary of the purpose of this pull request._
 _These tasks are for the pull request creator to tick off._
 
 - [ ] Developer testing
+- [ ] Make sure relevant changelog is updated if any changes affect users
```
Can you link to the changelogs in the PR description to make it more convenient? And can you move Developer Testing to the Reviewer Tasks here? (in all PR templates)

Craig-J replied:
Added. Can't test PR template relative links are correct without merge 😞

Craig-J committed:
Add missing changelog fragments, amend PR templates

Craig-J committed:
Add comment for new event

Craig-J committed:
Merge branch 'develop' into feature/web-tooling-changelog

Craig-J commented:
> Windows changelog is missing #410
> 
> BRS changelog is missing #412

Added #410 to changes to web tooling changelog. Nothing visible for users of main product yet so not mentioned in touchfree windows changelog.

Added #412 to brightsign changelog.
